frontend/public/components/dashboard/dashboards-page/overview-dashboard/status-card.tsx

```diff
@@ -167,6 +167,7 @@ const ClusterAlerts = withDashboardResources(
     const alerts = getAlerts(alertsResponse);
 
     const cv = _.get(resources.cv, 'data') as ClusterVersionKind;
+    const cvLoaded = _.get(resources.cv, 'loaded');
```

Comment:
We probably want to check the `CLUSTER_VERSION` flag before watching this resource.